be/btree.c

```diff
@@ -1893,10 +1893,10 @@ M0_INTERNAL void m0_btree_lrulist_set_lru_config(int64_t slow_lru_mem_release,
 	M0_ASSERT(lru_space_wm_high >= lru_space_wm_target &&
 		  lru_space_wm_target >= lru_space_wm_low);
 
-	M0_LOG(M0_INFO, "Btree LRU List Watermarks: Low - %"PRIi64" Mid - "
+	M0_LOG(M0_NOTICE, "Btree LRU List Watermarks: Low - %"PRIi64" Mid - "
```

Comment:
Do we need NOTICE here, we can keep it ALWAYS and when enabled NOTICE log level on console than move it to NOTICE

Reply:
These are one time logs printed when m0d is started. We can put this logs put as M0_NOTICE .